### ISSUE TITLE
Support of `help wanted` label

### ISSUE THREAD

**Issue by akosyakov** [musically-ut/first-timers-only-bot#6]
I would like to add support of `help wanted` label, but there are some many issues already: https://api.github.com/search/issues?q=label:"help%20wanted"+is:issue+is:open&sort=updated&order=desc

Will the bot post all of them at once? Is it alright?

**Comment by musically-ut** [comment]
The bot can be run with `--only-save` option to update the local DB:

https://github.com/musically-ut/first-timers-only-bot/blob/7438415d15e4a6f54666230a7aff46c35df44942/first_timers/run.py#L15-L17

Thereafter, it'll only report the new issues which appear with the label `help wanted`.

If you are interested in working on a PR for this, you may want to wait until I finish #4. Or why not work on #4 first? :)

**Comment by cg-cnu** [comment]
Hey guys, I am just wondering... 

The purpose of this bot I believe is to the find issues which are beginner friendly. The idea is to help beginners by taking the guess work out of their search process so that they can jump start and contribute. I just want all the issues reported by this bot to be simple enough. And flags like `help wanted` or even the existing `up for grabs` will have the regular issues which I feel will defeats the purpose. 

I am scouting the repos for beginners tasks when am starting out and found that beginner issues are also tagged with `good first contribution`, `bit-size`, `simple`, `easy`. I think these flags will fall under the category of this bot. 

These are just my views and am open to discussion. Please feel free to correct me.
Let me know what you guys think :)

**Comment by akosyakov** [comment]
@cg-cnu yeah, I've started labeling subset of `help wanted` with `beginners`, so it is fine with me to close it

**Comment by cg-cnu** [comment]
@musically-ut Since this is your project. I am looking for your opinion on this one. What do you say :smile:
